zebra-consensus/src/verify.rs

```diff
@@ -4,6 +4,7 @@
 //! verification.
 
 mod block;
+pub mod redjubjub;
```

Comment:
@hdevalence I removed the explicit `Item` and `Verifier` exports to preemptively avoid a collision with future verifiers like `Groth16` etc.